tools/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2022 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+  `java-platform`
+  `maven-publish`
+  signing
+  id("org.projectnessie.nessie-project")
+}
+
+extra["maven.name"] = "Nessie - Tools"
+
+javaPlatform { allowDependencies() }
```

Comment:
What does this do?

Reply:
This "simulates" the Maven parent-pom behavior. Wanted to make the pom artifacts look as much as the current build.

Reply:
TIL :+1: